src/index.html

```diff
@@ -8,10 +8,114 @@
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label for="surname">Surname:</label>
+        <input type="text" class="input"
+        id="surname"
+        name="surname"
+        autocomplete="off"
+        minlength="3"
+        required>
```

Comment:
```suggestion
        <input 
          type="text"
          class="input"
          id="surname"
          name="surname"
          autocomplete="off"
          minlength="3"
          required
        >
```
Форматируй теги тиаким образом